doc/pmon/pmon-chassis-requirements.md

```diff
@@ -0,0 +1,37 @@
+Section 1 Requirements that are needed by default:-
+        1. On LC the reboot command should power-cycle the entire LC . Expectation is Peer node should detect link down when reboot is given on LC
+		2. On RP the reboot command should reboot the entire system (RP and LC). . Expectation is Peer node should detect link down when reboot is given on RP
+		3. Config shut/unshut of LC will be supported as per the Chassis-d design.
```

Comment:
not all platforms can support LC shut as there might not be power control on LC

Reply:
@abdosi will update the point `from supervisor`.

Reply:
shut: power shut for card (platform dependent)
unshut: bring power back (platform dependent)

Reply:
reboot-cause shows what on LC when above option are invoke on supervisor.

Reply:
`config chassis startup <module-name>`
`config chassis shutdown <module-name>`
`reboot <module-name>`

module-name here are are LC. FC are not in discussion here.

Reply:
> reboot <module-name>
similar to this, there should be 'shutdown <module-name> CLI to shutdown specified module (LC/FC).

>config chassis startup <module-name>
>config chassis shutdown <module-name>
intent/goal of these commands  is to do config shutdown or startup (reload/bring-up) of specified module.
wondering as to why 'chassis' keyword is there in these commands?

Reply:
> 

In my opinion when a FC goes bad/down, expect syslog be generated.  For system that has extra FC redundancy I think losing one FC does not impact the overall operation of the chassis. So in this case the syslog should cause an alarm to allow schedule maintenance service to replace the FC. If loosing another FC or chassis has no FC redundancy then the lost of FC syslog should clearly indicate it is running in degraded mode with expected traffic impact.  This syslog should cause alarm service to detect and trigger mitigation steps (whether be Admin user intervention or automation to start isolate it). The chassis itself can not tell if there are redundancy built into the involved network and if trigger self isolation it might causes more customer impact...  This is just my own opinion...